ayekan/mimir/mimir.yaml

```diff
@@ -0,0 +1,197 @@
+# Mimir components
+mimir:
+  structuredConfig:
+    common:
+      storage:
+        backend: s3
```

Comment:
Looking at the `values.yaml` for the mimir chart, it doesn't appear that there is support for using a k8s secret for the s3 credentials. https://github.com/grafana/mimir/blob/main/operations/helm/charts/mimir-distributed/values.yaml

This is sub-optional as it make it difficult for us to use an external secrets operator.  For the moment, we are going to need to resort to interpolating a value from an env var into the yaml.